app/jobs/reports/combined_invoice_supplement_report.rb

```diff
@@ -54,9 +54,9 @@ def combine_by_iaa_month(by_iaa_results:, by_issuer_results:)
       by_issuer_iaa_issuer_year_months = by_issuer_results.
         group_by { |r| r[:iaa] }.
         transform_values do |iaa|
-          iaa.group_by { |r| r[:issuer] }.
-            transform_values { |issuer| issuer.group_by { |r| r[:year_month] } }
-        end
+        iaa.group_by { |r| r[:issuer] }.
+          transform_values { |issuer| issuer.group_by { |r| r[:year_month] } }
+      end
```

Comment:
Does the cop just not understand blocks?

I think it's indented the contents of this block with the outside of it, which is... very broken.

Is this maybe why we disabled this? Or is this maybe a smell that if it breaks rubocop's parser, the code could be tidied up another way?